### ISSUE TITLE
Super Knockback option in velocity

### ISSUE THREAD

**Issue by VirxEC** [ImpactDevelopment/ImpactIssues#1576]
**Is your feature request related to a problem? Please describe.**
It's annoying when you would like to knock someone back really far but there's no module.

**Describe the solution you'd like**
Under Velocity, add an option called "Super Knockback" or something that deals extra knockback to other players by abusing sprint mechanics. Also add a chance option for it.

**Describe alternatives you've considered**
Put the option as a module under combat.

**Additional context**
LiquidBounce(.net)x has a similar module.

**Comment by p1u70n** [comment]
Yeah, it's will be very nice to have this option in Impact (I was about to create a new feature request but I found this one).
